app/assets/stylesheets/welcome.scss.erb

```diff
@@ -354,3 +393,137 @@ nav.custom_nav{
 a.remove-text {
   color: #ffffff !important;
 }
+
+.about-top {
+  position: relative;
+  background-image: url("https://pixabay.com/static/uploads/photo/2014/09/07/17/41/girl-438129_960_720.jpg");
```

Comment:
It's not a good idea to hotlink images from a site you don't control. Better to download it and link it locally. Also for SEO purposes, the name of the image should be changed to tell us something about EventX and NOT pixabay.